src/com/yetanalytics/lrs_admin_ui/views/accounts.cljs

```diff
@@ -73,24 +73,24 @@
                  :class "icon-secret pointer"
                  :on-click #(swap! hide-pass not)}
              (str (cond
-                    @hide-pass "Show"
-                    :else "Hide"))]]
+                    @hide-pass @(subscribe [:lang/get :accounts.new.password.show])
+                    :else @(subscribe [:lang/get :accounts.new.password.hide])))]]
            [:li
             [copy-text
              {:text (:password new-account)
               :on-copy #(dispatch [:notification/notify false
-                                   "Copied New Password!"])}
-             [:a {:class "icon-copy pointer"} "Copy"]]]]]
+                                   @(subscribe [:lang/get :notification.accounts.password-copied])])}
+             [:a {:class "icon-copy pointer"} @(subscribe [:lang/get :accounts.new.password.copy])]]]]]
```

Comment:
Would having deeply-nested subscription derefs cause problems (like these subs in dispatch functions)? Even if not, I'd prefer if we extracted these subscriptions into let-bindings just to make the code less cluttered.

Reply:
If the few extra redraws aren't a problem then we can leave it as this. Don't be surprised though if I refactor all these calls into let-bindings the next time I touch the code.

Reply:
im not sure why you would but if you DO, then do it to 160 of them. Also keep in mind that lets will cause MORE redraw, not less.